packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.jsx

```diff
@@ -166,100 +230,120 @@ const BuySellModal = () => {
         if (!is_modal_open) {
             setErrorMessage(null);
         }
+
+        my_profile_store.setSelectedPaymentMethod('');
```

Comment:
**suggestion**: Could we refactor these 2 lines `234` and `235` out? I removed these lines from the buy-sell-modal since its already handled by the `ModalForm` component and it would affect saving the form state